dt/nova/nova02beta/edpm/nodeset/kustomization.yaml

```diff
@@ -0,0 +1,102 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+transformers:
+  # Set namespace to OpenStack on all namespaced objects without a namespace
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: _ignored_
+      namespace: openstack
+    setRoleBindingSubjects: none
+    unsetOnly: true
+    fieldSpecs:
+    - path: metadata/name
+      kind: Namespace
+      create: true
+
+components:
+  - ../../../../../lib/dataplane/nodeset
+
+resources:
+  - baremetalset-password-secret.yaml
+  - nova_sriov.yaml
+
+replacements:
```

Comment:
I believe this section misses spec.nodes and many other data provided in examples. When defined there, kustomize would not wire in anything but things listed here, right? I compared this to bmo01 DT

Reply:
TIL, thanks!